homework24/resources/utils.py

```diff
@@ -0,0 +1,20 @@
+import random
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.common.by import By
+
+
+def fill_field(wait, locator, value):
```

Comment:
Since, you are using typehints:
```suggestion
def fill_field(wait, locator: type?, value: type?) -> None:
```